util/file.py

```diff
@@ -473,3 +494,33 @@ def line_count(infname):
 def touch(fname, times=None):
     with open(fname, 'a'):
         os.utime(fname, times)
+
+def touch_empty(fname):
+    '''Make each file in fnames a zero-length file with the current timestamp.'''
+    with open(fname, 'w'):
+        pass
+
+def dump_file( fname, value ):
+    """store string in file"""
+    with open( fname, 'w' )  as out: 
+        out.write( str( value ) )
+
+def slurp_file( fname ):
```

Comment:
I'll add a default param for max file size to slurp; if you want to slurp a larger file you'll have to override the param explicitly.  I only use it when reading a file I know to be small, like a user-written config file; there it leads to clearer code.